apps/er-coap/er-coap-observe.c

```diff
@@ -58,9 +58,9 @@ LIST(observers_list);
 /*---------------------------------------------------------------------------*/
 /*- Internal API ------------------------------------------------------------*/
 /*---------------------------------------------------------------------------*/
-coap_observer_t *
+static coap_observer_t *
```

Comment:
When changing the signature of this function, the module prefix "coap_" should be removed. Furthermore, is there a need to not have the <code>const</code> qualifier set for the addr variable (and possibly doing the same for functions called from <code>coap_add_observer</code> that use <code>addr</code>?